x/programs/examples/storage/storage.go

```diff
@@ -29,23 +29,23 @@ func ProgramPrefixKey(id []byte, key []byte) (k []byte) {
 // Program
 //
 
-func ProgramKey(id ids.ID) (k []byte) {
-	k = make([]byte, 1+consts.IDLen)
-	copy(k[1:], id[:])
+func ProgramKey(address codec.Address) (k []byte) {
+	k = make([]byte, 1+codec.AddressLen)
+	copy(k[1:], address[:])
```

Comment:
Should probably leave a comment why you don't populate k[0]

Reply:
will leave a comment, i think we should explicitly set `k[0]` to `programPrefix` here to make it clear as well.